dd-sdk-android/src/test/kotlin/com/datadog/android/rum/resource/RumResourceInputStreamTest.kt

```diff
@@ -555,11 +555,11 @@ internal class RumResourceInputStreamTest {
 
     @Test
     fun `𝕄 register resource 𝕎 read() + mark() + reset() + close() {manual}`(
-        @StringForgery content: String
+        forge: Forge
     ) {
         // Given
+        val content = forge.anAlphabeticalString(size = forge.anInt(16, 1024))
         val contentBytes = content.toByteArray()
-        assumeTrue(contentBytes.size > 6)
         val inputStream = contentBytes.inputStream()
         testedInputStream = RumResourceInputStream(inputStream, fakeUrl)
         val mark = max(contentBytes.size / 3, 2)
```

Comment:
it seems like this line should be modified, because `min(contentBytes.size / 3) == 5`, so `max` call is redundant.